From a01d4b474c68f6acedc2b8bed7b2ea7cc0125f30 Mon Sep 17 00:00:00 2001
From: Giuseppe Scrivano <gscrivan@redhat.com>
Date: Thu, 29 May 2014 12:06:08 +0200
Subject: [PATCH 2/2] osdict: replace "and" with "or"

In the operating systems list we use both "and later|earlier" and "or
later|earlier" forms.  Stick to one.

RHEL-only

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>
---
 virtinst/osdict.py | 34 +++++++++++++++++-----------------
 1 file changed, 17 insertions(+), 17 deletions(-)

diff --git a/virtinst/osdict.py b/virtinst/osdict.py
index 62c41c1..336f741 100644
--- a/virtinst/osdict.py
+++ b/virtinst/osdict.py
@@ -268,7 +268,7 @@ OS_TYPES = {
         }
     },
     "rhel7": {
-        "label": "Red Hat Enterprise Linux 7 and later",
+        "label": "Red Hat Enterprise Linux 7 or later",
         "distro": "rhel",
         "supported": True,
         "devices" : {
@@ -383,7 +383,7 @@ OS_TYPES = {
         }
     },
     "fedora18": {
-        "label": "Fedora 18 and later",
+        "label": "Fedora 18 or later",
         "distro": "fedora",
         "supported": True,
         "devices" : {
@@ -403,7 +403,7 @@ OS_TYPES = {
         },
     },
     "opensuse12": {
-        "label": "openSuse 12 and later",
+        "label": "openSuse 12 or later",
         "distro": "suse",
         "supported": True,
         "devices" : {
@@ -418,7 +418,7 @@ OS_TYPES = {
         "supported": True,
     },
     "sles11": {
-        "label": "Suse Linux Enterprise Server 11 and later",
+        "label": "Suse Linux Enterprise Server 11 or later",
         "distro": "suse",
         "supported": True,
         "devices" : {
@@ -428,11 +428,11 @@ OS_TYPES = {
     },
 
     "mandriva2009": {
-        "label": "Mandriva Linux 2009 and earlier",
+        "label": "Mandriva Linux 2009 or earlier",
         "distro": "mandriva"
     },
     "mandriva2010": {
-        "label": "Mandriva Linux 2010 and later",
+        "label": "Mandriva Linux 2010 or later",
         "distro": "mandriva",
         "devices" : {
             DISK : VIRTIO_DISK,
@@ -445,7 +445,7 @@ OS_TYPES = {
         "distro": "mandriva",
     },
     "mes5.1": {
-        "label": "Mandriva Enterprise Server 5.1 and later",
+        "label": "Mandriva Enterprise Server 5.1 or later",
         "distro": "mandriva",
         "supported": True,
         "devices" : {
@@ -455,7 +455,7 @@ OS_TYPES = {
     },
 
     "mageia1": {
-        "label": "Mageia 1 and later",
+        "label": "Mageia 1 or later",
         "distro": "mageia",
         "supported": True,
         "devices" : {
@@ -493,7 +493,7 @@ OS_TYPES = {
         }
     },
     "debianwheezy": {
-        "label": "Debian Wheezy and later",
+        "label": "Debian Wheezy or later",
         "distro": "debian",
         "sortby": "debian7",
         "supported": True,
@@ -580,7 +580,7 @@ OS_TYPES = {
         },
     },
     "ubuntuquantal": {
-        "label": "Ubuntu 12.10 (Quantal Quetzal) and later",
+        "label": "Ubuntu 12.10 (Quantal Quetzal) or later",
         "distro": "ubuntu",
         "supported": True,
         "devices" : {
@@ -646,7 +646,7 @@ OS_TYPES = {
         "distro": "winserv",
     },
     "win2k8": {
-        "label": "Microsoft Windows Server 2008 and later",
+        "label": "Microsoft Windows Server 2008 or later",
         "supported": True,
         "sortby": "mswinserv2008",
         "distro": "winserv",
@@ -658,7 +658,7 @@ OS_TYPES = {
         "distro": "win",
     },
     "win7": {
-        "label": "Microsoft Windows 7 and later",
+        "label": "Microsoft Windows 7 or later",
         "supported": True,
         "sortby": "mswin7",
         "distro": "win",
@@ -677,13 +677,13 @@ OS_TYPES = {
         "label": "Sun Solaris 9",
     },
     "solaris10": {
-        "label": "Sun Solaris 10 and later",
+        "label": "Sun Solaris 10 or later",
         "devices" : {
             INPUT : USB_TABLET,
         },
     },
     "opensolaris": {
-        "label": "Sun OpenSolaris and later",
+        "label": "Sun OpenSolaris or later",
         "devices" : {
             INPUT : USB_TABLET,
         },
@@ -710,7 +710,7 @@ OS_TYPES = {
         },
     },
     "freebsd8": {
-        "label": "FreeBSD 8.x and later" ,
+        "label": "FreeBSD 8.x or later" ,
         "supported": True,
         "devices" : {
             NET : { "model" : [ (HV_ALL, "e1000") ] }
@@ -718,7 +718,7 @@ OS_TYPES = {
     },
 
     "openbsd4": {
-        "label": "OpenBSD 4.x and later" ,
+        "label": "OpenBSD 4.x or later" ,
         # http://calamari.reverse-dns.net:980/cgi-bin/moin.cgi/OpenbsdOnQemu
         # https://www.redhat.com/archives/et-mgmt-tools/2008-June/msg00018.html
         "devices" : {
@@ -746,7 +746,7 @@ OS_TYPES = {
         "label": "Novell Netware 5",
     },
     "netware6": {
-        "label": "Novell Netware 6 and later",
+        "label": "Novell Netware 6 or later",
         "pv_cdrom_install": True,
     },
 
-- 
1.9.3

